src/components/Modal/Modal.tsx

```diff
@@ -31,6 +31,13 @@ export interface ModalProps {
   api?: MutableRefObject<ModalApi | undefined>;
   /** Adds a border for the header. */
   drawHeaderBorder?: boolean;
+  /**
+   * Hides `x` icon and removes ability to close modal by clicking outside of it.
+   *
+   * Relies on you to provide a way to close the modal, i.e. a cancel or confirm button.
+   * Useful if you definitely need to force the user to make a choice.
+   * */
+  forceModalInteraction?: boolean;
```

Comment:
Thought: I think my first guess at naming this would have been `allowClosing` (defaults true, users can set it to false)... Maybe still a slight preference for that, but not totally against `forceModalInteraction`; wdyt?

Reply:
Names are hard, and i cant come up with anything better 🙃 Will update to `allowClosing`